src/thor/costmatrix.cc

```diff
@@ -777,6 +782,8 @@ void CostMatrix::SetSources(GraphReader& graphreader,
 
       // Set the initial not_thru flag to false. There is an issue with not_thru
       // flags on small loops. Set this to false here to override this for now.
+      // NOTE(nils): shouldn't not_thru_pruning be set to the opposite of the edge's not_thru flag?
```

Comment:
this note is redundant. it doesn't matter really, constant `false` is as good as dynamic since we force the `not_thru` flag on the correlated edges anyways